src/eduid_webapp/eidas/views.py

```diff
@@ -112,6 +113,10 @@ def assertion_consumer_service():
         current_app.logger.debug('Auth response:\n{!s}\n\n'.format(authn_response))
         current_app.logger.debug('Session info:\n{!s}\n\n'.format(session_info))
 
+        # Remap nin in staging environment
+        if os.environ.get('ENVIRONMENT', None) == 'staging':
```

Comment:
Shouldn't self.app.config['ENVIRONMENT'] be used here? That is set to either 'staging' or 'production' in our current environments...

Reply:
Sounds good